src/ApparentHorizons/StrahlkorperInDifferentFrame.cpp

```diff
@@ -232,11 +234,21 @@ void strahlkorper_in_different_frame(
   const double padding = 0.10;
   get(bracket_r_min) = r_min * (1.0 - padding);
   get(bracket_r_max) = r_max * (1.0 + padding);
-  RootFinder::bracket_possibly_undefined_function_in_interval(
-      make_not_null(&get(bracket_r_min)), make_not_null(&get(bracket_r_max)),
-      make_not_null(&get(f_bracket_r_min)),
-      make_not_null(&get(f_bracket_r_max)), radius_function_for_bracketing);
-
+  try {
+    RootFinder::bracket_possibly_undefined_function_in_interval(
+        make_not_null(&get(bracket_r_min)), make_not_null(&get(bracket_r_max)),
+        make_not_null(&get(f_bracket_r_min)),
+        make_not_null(&get(f_bracket_r_max)), radius_function_for_bracketing);
+  } catch (std::runtime_error& e) {
+    ERROR(
```

Comment:
With the change to exception handling you should be able to add tests that check the errors your adding. Eg `CHECK_THROWS_WITH`. Could that be done fairly easily? The other errors your adding/improving too? We no longer need a separate `ERROR_TEST` so I think it's good to test error messages whenever possible :)

Reply:
I added a CHECK_THROWS_WITH in Test_RootBracketing.cpp

It is harder to add a CHECK_THROWS_WITH in Test_StrahlkorperInDifferentFrame because I would need to contrive a case where the radius of the mapped Strahlkorper at a particular grid point is within 1e-8 of the excision boundary but outside of it.

Reply:
Okay, sounds good!